doc/events/http_check.md

```diff
@@ -0,0 +1,28 @@
+---
+module: [kind=event] http_check
+see: http.checkURLAsync To check a URL asynchronously.
+---
+
+The @{http_check} event is fired when a URL check finishes.
+
+This event is normally handled inside @{http.checkURL}, but it can still be seen when using @{http.checkURLAsync}.
+
+## Return Values
+1. @{string}: The URL requested to be checked.
+2. @{boolean}: Whether the check succeeded.
+3. @{string|nil}: If the check failed, a reason explaining why the check failed.
+
+## Example
+Checks a valid URL:
+```lua
+http.checkURLAsync("http://www.example.com")
```

Comment:
Writing examples for these is hard, because `checkURLAsync` has a somewhat ugly interface.

More precisely, ` http.checkURLAsync("ht:/")` _won't_ queue a `http_check` event, as it's a syntactically incorrect URL - it just returns `false, error`. `http_check` events are only queued when the allowed/denied check complete (it's asynchronous as it requires a DNS lookup).

I'd honestly just leave these examples off for now. It's such a useless event (who bulk checks URLs?!) and any actual example is just going to end up looking like the source for `http.checkURL`.